composer.json

```diff
@@ -33,12 +46,14 @@
     "wp-phpunit/wp-phpunit": "^6.5",
     "yoast/phpunit-polyfills": "^4.0",
     "phpstan/php-8-stubs": "^0.4.0",
-    "phpstan/phpstan-strict-rules": "^1.6"
+    "phpstan/phpstan-strict-rules": "^1.6",
+    "wpackagist-plugin/plugin-check": "^1.5"
```

Comment:
I wasn't entirely sure how to install the sniffs for PCP, so I installed them via WPackagist. @swissspidy is this right?

Reply:
Interesting! I suppose so 🙂 

When we added those custom sniffs we haven't really thought about making these available to others. But if it works, it works.